R/new_ticker.R

```diff
@@ -0,0 +1,33 @@
+new_ticker <- function(label, n, progress = NA) {
+  # if progress bar should'nt be shown, return identity
+  # message or fail depending on `progress`
+
+  if (is.na(progress)) {
+    progress <- interactive()
+    if (progress && !requireNamespace("progress")) {
+      message("The 'progress' package must be installed in order to display progress bars.")
```

Comment:
This package uses rlang idioms where approriate.

```suggestion
    progress <- is_interactive()
    if (progress && !is_installed("progress")) {
      inform("The 'progress' package must be installed in order to display progress bars.")
```